Maquetado/scss/details.scss

```diff
@@ -94,40 +112,77 @@
 }
 
 .book-title {
+  align-self: end;
+  grid-area: title;
   @include text-format(#{$big-font-size + 12}, $font-weight, #{$line-height-base + 18});
+  justify-self: center;
   margin-right: 5px;
 }
 
 .book-gender {
+  align-self: start;
+  grid-area: gender;
   @include text-format(#{$big-font-size + 4}, $font-weight, #{$line-height-base + 8});
+  justify-self: center;
 }
 
 .item-key {
-  @include text-format($big-font-size, $font-weight, $line-height-base);
+  @include text-format(#{$big-font-size + 4}, $font-weight, $line-height-base);
   margin-right: 5px;
 }
 
 .item-value {
-  font-size: $big-font-size;
+  font-size: #{$big-font-size + 4};
   line-height: #{$line-height-base + 4};
 }
 
 .list {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-gap: 50px;
+  grid-template: repeat(2, 275px) / repeat(4,165px);
+  grid-template-areas: 'a''b''c''d''e''f''g''h';
   justify-content: center;
-  margin: 0 auto;
-  max-width: 1000px;
-  width: 100%;
+  margin: 20px auto;
 }
 
-.list-book {
-  background-color: $white;
-  display: flex;
-  flex-flow: column wrap;
-  @include shadow;
-  margin: 20px;
-  padding: 25px;
+.list-book-a {
+  @extend %list-book;
```

Comment:
You don't need extend this in all, you can define a general class for this cards and use `nth-child` for assign `grid-area` property